3.3-apache/Dockerfile

```diff
@@ -5,10 +5,26 @@ LABEL maintainer="Chaim Sanders <chaim.sanders@gmail.com>"
 ARG COMMIT=v3.3/dev
 ARG BRANCH=v3.3/dev
 ARG REPO=SpiderLabs/owasp-modsecurity-crs
-ENV WEBSERVER=Apache
-ENV PARANOIA=1
-ENV ANOMALYIN=5
-ENV ANOMALYOUT=4
+
+ENV PARANOIA=1 \
+    ANOMALY_INBOUND=5 \
+    ANOMALY_OUTBOUND=4 \
+    TIMEOUT=60 \
```

Comment:
`TIMEOUT` is too generic, I fear. This makes the parameter hard to understand.

This is called `APACHE_TIMEOUT` in [our VSHN image](https://github.com/vshn/modsecurity-docker#apache), which explains its function more clearly. Should we call it `PROXY_TIMEOUT` or `WEBSERVER_TIMEOUT`?

If this is hard to abstract, as for the wording, we may stick to `APACHE_TIMEOUT` and related `NGINX_*` variables, if needed. Hopefully that's not necessary.

Reply:
I'd vote for `APACHE_TIMEOUT` since I don't think there is an equivalent config for nginx.

https://httpd.apache.org/docs/2.4/mod/core.html#timeout

Reply:
It's hard to abstract because it does SO MANY DIFFERENT THINGS according to the docs...

I'd propose `APACHE_TIMEOUT` and then maybe have a generic `PROXY_TIMEOUT` which sets [`ProxyTimeout`](https://httpd.apache.org/docs/2.4/mod/mod_proxy.html#proxytimeout)?

Reply:
The proxy timeout will be set up in a future update. We have` PROXY_TIMEOUT` directive in `apache-vhosts.conf ` and the equivalent `proxy_connect_timeout` for nginx.